index.js

```diff
@@ -2,19 +2,23 @@ const isString = x => typeof x === 'string'
 const isArray = x => Array.isArray(x)
 const isObject = x => typeof x === 'object' && !isArray(x)
 
-const clean = (a, b) => (!b ? a : isString(b[0]) ? a.concat([b]) : a.concat(b))
-const child = x => f =>
-  x || x === 0 ? (isArray(x) ? x.reduce(clean, []).map(f) : x + '') : []
+const clean = (arr, n) =>
+  (n && Array.prototype.push.apply(arr, (isString(n[0]) ? [n] : n)), arr)
 
-const build = (x, y, z) => node =>
-  !!node && isString(node)
+const child = (n, cb) =>
+  n !== undefined
+    ? (isArray(n)
+      ? n.reduce(clean, []).map(cb)
+      : n + '')
+    : []
+
+export const h = (x, y, z) => node =>
```

Comment:
I had some issues before with bundling exported recursive functions properly so I think the whole `export const h = build` was from that. Now we are not bundling I don't think it will be an issue. So :+1:

Reply:
I wondered that, and I guessed that since 3rd party app would bundle / compress the final code, maybe this lib didn't need to do so...